src/ic-cdk/src/api/management_canister/http_request/mod.rs

```diff
@@ -0,0 +1,100 @@
+//! Canister HTTP request.
```

Comment:
A suggestion for future work when moving a file like this.  Open a separate, preparatory PR that renames (as in this case) `request.rs -> request/mod.rs`.  This will make it easier / possible to review the differences.  As is, it's not reasonable for a reviewer to review the changes: they must review the whole file, or manually diff the files.